### ISSUE TITLE
android version

### ISSUE THREAD

**Issue by fukemy** [DlhSoftTeam/Ganttis#4]

**Comment by DlhSoftTeam** [comment]
Currently we don't have an Android version of this package. For Android, however, you can try using our JavaScript based GanttChartView in a WebView. The mentioned component is available with [DlhSoft Gantt Chart Hyper Library](https://dlhsoft.com/GCHL), sold separately.

**Comment by suleimanovs** [comment]
Will this variant really work in android? WebView with gantt?

**Comment by DlhSoftTeam** [comment]
Gantt Chart Hyper Library, a different product, allows you to display an interactive Gantt chart in the JavaScript world. Android allows JavaScript code in a WebView. Windows, for example, does the same under UWP's WebView. We did test it under UWP (Windows apps) and it works - we have a product for that technology too based on this integration being possible (Gantt Chart Modern Library).

These being said, we did NOT test it under Android though, so it will be up to you to check it out and implement, like with any other JavaScript code, if you want to go that route.

Note: Ganttis framework is different from Gantt Chart Hyper Library in many ways. Ganttis will not run on Android.
